File: test/com/williambarkoff/csx/rustremoval/GettingStartedTest.java
package com.williambarkoff.csx.rustremoval;

import org.junit.Test;

import static org.hamcrest.CoreMatchers.is;
import static org.junit.Assert.assertThat;

public class GettingStartedTest {
    @Test
    public void add() {
        assertThat(GettingStarted.add(2, 3), is(5));
    }
}

File: test/com/williambarkoff/csx/rustremoval/GettingStartedTest.kt
package com.williambarkoff.csx.rustremoval

import org.junit.Test

import org.hamcrest.core.Is.`is`
import org.junit.Assert.*

class GettingStartedTest {

    @Test
    fun add() {
        assertThat(GettingStarted.add(2, 3), `is`(5))
    }
}